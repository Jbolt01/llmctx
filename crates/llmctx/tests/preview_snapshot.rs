use insta::assert_snapshot;

#[test]
fn preview_default_renders() {
    let rendered = "```
fn main() {
    println!(\"hello\");
}
```";
    assert_snapshot!("preview_default", rendered);
}
