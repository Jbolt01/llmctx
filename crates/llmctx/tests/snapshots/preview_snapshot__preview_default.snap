---
source: crates/llmctx/tests/preview_snapshot.rs
expression: rendered
---
```
fn main() {
    println!("hello");
}
```
